wrapanapi/systems/ec2.py

```diff
@@ -809,6 +812,10 @@ def delete_s3_bucket(self, bucket_name):
         bucket = self.s3_connection.Bucket(bucket_name)
         self.logger.info("Trying to delete bucket '%s'", bucket_name)
         try:
+            for key in bucket.objects.all():
+                response = key.delete()
+                self.logger.info("Success: key '%s' associated with bucket '%s' was deleted",
```

Comment:
This is actually implementing the TODO in the method's docblock, which should be removed.

This action should be an option passed to delete_s3_bucket though, I think defaulting to `False` so that if there are objects in the bucket delete will still fail.